DiscordChatExporter.Core/Discord/Data/Message.cs

```diff
@@ -27,14 +27,18 @@ public partial record Message(
     IReadOnlyList<User> MentionedUsers,
     MessageReference? Reference,
     Message? ReferencedMessage,
-    Interaction? Interaction
+    Interaction? Interaction,
+    MessageSnapshot? ForwardedMessage
 ) : IHasId
 {
     public bool IsSystemNotification { get; } =
         Kind is >= MessageKind.RecipientAdd and <= MessageKind.ThreadCreated;
 
     public bool IsReply { get; } = Kind == MessageKind.Reply;
 
+    // A message is a forward if its reference type is Forward
+    public bool IsForward { get; } = Reference?.Kind == MessageReferenceKind.Forward;
```

Comment:
```suggestion
    public bool IsForwarded { get; } = Reference?.Kind == MessageReferenceKind.Forward;
```

And maybe move it after `IsReplyLike`

Reply:
Change made.